nncf/common/quantization/quantizer_setup.py

```diff
@@ -28,7 +28,11 @@
 QuantizationPointId = int
 
 DEFAULT_QUANTIZER_CONFIG = QuantizerConfig(
-    num_bits=8, mode=QuantizationMode.SYMMETRIC, signedness_to_force=None, per_channel=False
+    num_bits=8,
```

Comment:
Can we make only one entity with this information? I see that `DEFAULT_QUANTIZATION_TYPES` from the `nncf/common/quantization/quantizer_propagation/solver.py` (file above) contains exact same value.